src/frontend/css/tabs.css

```diff
@@ -16,7 +19,7 @@
   border: 1px solid #666666;
   border-radius: 5px 5px 0 0;
   border-bottom: none;
-  font-size: 1.5rem;
+  font-size: 24px;
```

Comment:
Any reason for this change? With the current default text size for our frontend, these are equivalent

Reply:
IIRC it's a mistake that this change slipped into the PR, but I saw there are `em`, `rem`, and `px` units in the CSS with some inconsistency in which is used in which context. (e.g. text is sized in `px, `em` and `rem` in different places. I know `em` and `rem` have different uses.)

At the time I was wondering if we could simplify it down to as few units as we could and make them more consistent. The thinking was this would make it a little easier to reason about elements in relation to eachother. But it's a non-issue right now, and maybe it's fine as it is.